exporters/stdout/stdoutmetric/config.go

```diff
@@ -74,6 +81,21 @@ func WithEncoder(encoder Encoder) Option {
 	})
 }
 
+// WithWriter sets the export stream destination.
+// If `WithEncoder` is also used, this option will have no effect.
```

Comment:
Not really. Depends which is set as the last 😉 I would say that it overrides previous `[WithEncoder]` and `[WithWriter]` options.